### DISCUSSION TITLE
#31. VATZ biweekly meeting at 2023-04-12 08:00:00 -05:00:00

### DISCUSSION THREAD

**Discussion by xellos00** [dsrvlabs/vatz#424] [General]
### 1. Overall
<img width="1212" alt="image" src="https://user-images.githubusercontent.com/6308023/228671175-c4706c8c-3b08-4ed1-a646-8d361d39bbbd.png">

### 2. Statistic Rate
Sprint | Issue fulfillment | progress rate(%)
--: | :--: | :--:
29 | 6/23 |  26
30 | 2/19 |  10
---

### 3. Previous 2 weeks Traffics

<img width="1264" alt="image" src="https://user-images.githubusercontent.com/6308023/231453756-2adcb6cc-ec37-4a23-a549-1a74976c4d57.png">

**Comment by xellos00** [comment]
# 동요


- https://github.com/dsrvlabs/vatz/issues/388
- https://github.com/dsrvlabs/vatz/issues/415
- https://github.com/dsrvlabs/vatz/issues/426

**Comment by rootwarp** [comment]
# 준교
- #419 
- #421 
- #426

**Comment by heejin-github** [comment]
## 희진
- #423

**Comment by gnongs** [comment]
## 진홍
- no issue

**Comment by skonhwang** [comment]
# 승곤
- no issue

**Comment by kim201212** [comment]
## 형기
- no issue
